experiments/AMIP/components/atmosphere/climaatmos_init.jl

```diff
@@ -77,9 +77,8 @@ function atmos_init(::Type{FT}, atmos_config_dict::Dict) where {FT}
     # set initial fluxes to zero
     @. integrator.p.precomputed.sfc_conditions.ρ_flux_h_tot = Geometry.Covariant3Vector(FT(0.0))
     @. integrator.p.precomputed.sfc_conditions.ρ_flux_q_tot = Geometry.Covariant3Vector(FT(0.0))
-    @. integrator.p.precomputed.sfc_conditions.ρ_flux_uₕ.components =
-        zeros(axes(integrator.p.precomputed.sfc_conditions.ρ_flux_uₕ.components))
-    parent(integrator.p.radiation.ᶠradiation_flux) .= parent(zeros(axes(integrator.p.radiation.ᶠradiation_flux)))
+    @. integrator.p.radiation.ᶠradiation_flux = Geometry.WVector(FT(0))
+    integrator.p.precomputed.sfc_conditions.ρ_flux_uₕ.components .= Ref(SMatrix{1, 2}([FT(0), FT(0)]))
```

Comment:
without `Ref`, this fails with `LoadError: MethodError: no method matching static_check_broadcast_shape(::ClimaCore.Spaces.SpectralElementSpace2D{…}, ::Tuple{…})`

Reply:
Okay, maybe revert this change and we can think about it in the future.

Reply:
I'm going to keep it as `Ref(SMatrix...)` because what we had before wasn't GPU compatible. But I agree we should change it in the future

Reply:
I am not sure that `Ref(SMatrix)` is correct. What we need is multiple separate instances of `SMatrix(zero)`, not multiple views over the same one.

Could you please check that using `Ref(SMatrix...)` does that?

Reply:
After using `Ref(SMatrix...)` to set the field, I get:
```
(parent(field))[:, :, :, 1] === (parent(field))[:, :, :, 2] = false
(parent(field))[:, :, :, 1] == (parent(field))[:, :, :, 2] = true
```
so it looks like they're separate instances